recipes/full_finetune_single_device.py

```diff
@@ -611,12 +587,7 @@ def train(self) -> None:
         The core training loop. Supports training on subsets of the dataset using the
         ``max_steps_per_epoch``.
         """
-        if self._compile:
-            self._logger.info(
-                "NOTE: torch.compile is enabled and model is compiled in first forward. Expect a relatively slow first iteration."
-            )
```

Comment:
Removing this is not strictly related, right? I'm not opposed, but we should at least log if we're compiling somewhere (I think we already do it in `compile_model`?)

Reply:
We already do this in compile model yeah.